lib/wickedbase.pm

```diff
@@ -258,8 +292,8 @@ sub setup_tunnel {
     my $local_ip = $self->get_ip(type => 'host');
     my $remote_ip = $self->get_remote_ip(type => 'host');
     my $tunnel_ip = $self->get_ip(type => $type);
-    file_content_replace($config, local_ip => $local_ip, remote_ip => $remote_ip, tunnel_ip => $tunnel_ip);
-    $self->wicked_command('ifup', $type);
+    file_content_replace($config, local_ip => $local_ip, remote_ip => $remote_ip, tunnel_ip => $tunnel_ip, iface => iface());
+    $self->wicked_command('ifup', 'all');
```

Comment:
I would discuss this with Marius . It seems to me it is two different test cases `ifup eth0` vs. `ifup all`

Reply:
yes, when you look into the legacy tests, there are actually 2 ways:
1) ifup eth0
    ifup tun1 (or whatever)
2) ifup all

not sure what is best here... I just took 'all' to make it simpler (1 line)

Reply:
looks like we agreed to have ifup all